tests/test_iqm_job.py

```diff
@@ -255,3 +257,20 @@ def test_result_with_timestamps(job, iqm_result_two_registers, iqm_metadata_with
     assert job.metadata.pop('timestamps') == iqm_metadata_with_timestamps.get('timestamps')
     assert 'timestamps' in result._metadata
     assert result.timestamps == iqm_metadata_with_timestamps.get('timestamps')
+
+
+def test_job_monitor(job, iqm_metadata):
+    client_result = RunResult(status=Status.READY, metadata=iqm_metadata)
+    run_responses = [
```

Comment:
There is no assertions in the test, please, add what is actually tested, e.g . just "job_monitor should not raise" or we actually want to verify contents of `monitor_string`? As far as I can see, it would be enough to test handling of every possible `Status ` and not to care about the monitoring string:

```suggestion
    run_responses = [RunStatus(status=s.value) for s in Status]
```